sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/DistributeConstants.java

```diff
@@ -106,6 +111,16 @@
      */
     static final String PARAMETER_UPDATE_SETUP_FAILED = "update_setup_failed";
 
+    /**
+     * API parameter for install identifier.
+     */
+    static final String PARAMETER_INSTALL_ID = "install_id";
+
+    /**
+     * API parameter for last installed release hash.
+     */
+    static final String PARAMETER_LAST_DOWNLOADED_RELEASE_HASH = "downloaded_release_hash";
```

Comment:
Why do we report release hash? there is a one to many relationship between a single release hash and multiple release ids. Release hash is just based on packagename and version, you can have multiple releases with same hash very easily.

Reply:
Will remove it, the description is legacy and misleading.